tests/msim/MSIM_PYACTION_CHANGING_SCHEDULE_ACTIONX_CALLBACK.DATA

```diff
@@ -0,0 +1,501 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2015 Statoil
+
+-- This simulation is based on the data given in 
+-- 'Comparison of Solutions to a Three-Dimensional
+-- Black-Oil Reservoir Simulation Problem' by Aziz S. Odeh,
+-- Journal of Petroleum Technology, January 1981
+--
+-- Here in this case the deck has been modified by adding a PYACTION and ACTIONX
+-- keyword. The deck is used in a integration test with the dummy simulator
+-- msim.
```

Comment:
I think you should mention the additional changes here. Currently the comment is the same as in MSIM_PYACTION_CHANGING_SCHEDULE.DATA